notebooks/samples/optimizer/ai_platform_optimizer_conditional_parameters.ipynb

```diff
@@ -0,0 +1,902 @@
+{
```

Comment:
<p>Remove the pi symbol next to Dataset</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/108/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>